files/angular-base-apps/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "angular-base-apps",
+  "repo": "base-apps/angular-base-apps",
+  "files": {
+    "include": ["dist/css/base-apps.css", "dist/css/base-apps.min.css", "dist/js/base-apps.js", "dist/js/base-apps.min.js", "dist/js/base-apps-templates.js", "dist/js/base-apps-templates.min.js"]
```

Comment:
You would need to set `basePath` to dist/ and then change the `include` to `./css/base-apps.css` to retain the directory structure.